aiida/backends/tests/cmdline/commands/test_computer.py

```diff
@@ -0,0 +1,106 @@
+"""Tests for the 'verdi computer' command."""
+import os
+import subprocess as sp
+from click.testing import CliRunner
+
+from aiida.backends.testbase import AiidaTestCase
+from aiida.cmdline.commands.computer import (disable_computer, enable_computer)
+from aiida.common.exceptions import NotExistent
+
+
+# pylint: disable=missing-docstring
+class TestVerdiComputerCommands(AiidaTestCase):
+    """Testing verdi computer commands.
+
+    Testing everything besides `computer setup`.
+    """
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
```

Comment:
the base `AiidaTestCase` already provides a `computer` class property. If this computer is just a base stud, maybe you could that one to avoid duplication